package.json

```diff
@@ -42,8 +42,8 @@
   },
   "private": true,
   "dependencies": {
-    "@nrwl/tao": "16.0.0",
-    "@nx/devkit": "16.0.0",
+    "@nrwl/tao": "16.8.1",
+    "@nx/devkit": "^16.8.1",
```

Comment:
Added here caret `^` so downstream users don't need to depend on specific version